# Fix CMake minimum version for coredumper
# CMake 3.27+ deprecated versions < 3.5, CMake 4.0.0 removed compatibility
--- CMakeLists.txt.orig	2025-11-01 10:00:00.000000000 +0000
+++ CMakeLists.txt	2025-11-01 10:00:00.000000000 +0000
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION  2.8.12)
+cmake_minimum_required(VERSION  3.5)
 if (POLICY CMP0048)
   cmake_policy(SET CMP0048 NEW)
 endif(POLICY CMP0048)
